cypress/commons/utils/apiUtils.js

```diff
@@ -216,8 +216,10 @@ const interceptUpdateScenario = (options) => {
   const alias = forgeAlias('reqUpdateScenario');
   cy.intercept({ method: 'PATCH', url: API_REGEX.SCENARIO, times: 1 }, (req) => {
     if (options?.validateRequest) options?.validateRequest(req);
+    const now = Date.now();
+    const userEmail = stub.getUser()?.email ?? 'stub.user@example.com';
```

Comment:
We usually use `dev.sample.webapp@example.com` as email of the service account user 🙂